python/tvm/relay/qnn/op/qnn.py

```diff
@@ -19,14 +19,15 @@
 
 from __future__ import absolute_import as _abs
 from . import _make
+from tvm import relay
 
 def requantize(data,
                input_scale,
                input_zero_point,
                output_scale,
                output_zero_point,
                rounding="TONEAREST",
-               out_dtype="int8"):
+               out_dtype='int8'):
```

Comment:
Is this really an unrelated change ?

Reply:
Yes, this was unintended change. I will be consistent in using single or double quotes.